lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -795,16 +793,20 @@ class StandardAccountTest {
                 Felt(3),
                 balanceContractAddress,
                 selectorFromName("increase_balance"),
+                Felt(0),
                 Felt(3),
-                Felt(10),
-                Felt(20),
-                Felt(30),
                 Felt(999),
                 selectorFromName("empty_calldata"),
+                Felt(3),
                 Felt(0),
                 Felt(123),
                 selectorFromName("another_method"),
+                Felt(3),
                 Felt(2),
+                Felt(5),
+                Felt(10),
+                Felt(20),
+                Felt(30),
```

Comment:
`expectedCalldata` was changed here, because `cairoVersion` which was passed before (Cairo 1) is different than the one which is now automatically determined (Cairo 0). 
`expectedCalldata` scheme before was based on https://github.com/software-mansion/starknet-jvm/blob/f43b8b59674cb1a8347753413443a1836b9deea5/lib/src/main/kotlin/com/swmansion/starknet/data/types/Execution.kt#L98-L112
now it's based on 
https://github.com/software-mansion/starknet-jvm/blob/f43b8b59674cb1a8347753413443a1836b9deea5/lib/src/main/kotlin/com/swmansion/starknet/data/types/Execution.kt#L114-L133

Reply:
Would it be possible to use Cairo 1 account in this test instead?

Reply:
Yup, we can. In fact I've updated `accountContractClassHash` in `DevnetClient`, and this way we have a Cairo 1 account.